hbase-rsgroup/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupAdminServer.java

```diff
@@ -193,76 +198,104 @@ private void checkServersAndTables(Set<Address> servers, Set<TableName> tables,
     }
   }
 
+  private enum MoveType {
+    TO, FROM
+  }
+
   /**
-   * Move every region from servers which are currently located on these servers,
-   * but should not be located there.
+   * When move a table to a group, all regions of it must be moved to group servers (to the group).
```

Comment:
I think you should make this description clearer regarding FROM , TO, TABLE and GROUP

Reply:
OK.